### PR TITLE
sshguard: update to 2.3.0

### PR DESCRIPTION
**Pull request by Schamschula**
add myself as co-maintainer
See: https://trac.macports.org/ticket/54524
See: https://trac.macports.org/ticket/57827

###### Type(s)

- [x] update

###### Tested on
macOS 10.13.6 17G4015
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: net/sshguard/Portfile [hunk 1]
```diff
@@ -3,11 +3,12 @@
 PortSystem          1.0
 PortGroup           bitbucket 1.0
 
-bitbucket.setup     sshguard sshguard 1.6.4 v
+bitbucket.setup     sshguard sshguard 2.3.0 v
+revision            0
```
Comment (pmetzger) [inline net/sshguard/Portfile]: Why the new `revision 0`?

**Comment by Schamschula** [comment]
Perry,

There was a thread on the mailing list, where it was agreed to always set the revision in the Portfile:

See: https://lists.macports.org/pipermail/macports-dev/2018-December/039919.html

**Comment by pmetzger** [comment]
@Schamschula This is what I get for not paying attention over Christmas. Yes, it seems reasonable to me, too.

**Comment by pmetzger** [comment]
LGTM, you should commit.

**Pull request merged**